lib/core/src/payjoin/pset/tests.rs

```diff
@@ -1,3 +1,4 @@
+#[allow(clippy::module_inception)]
 #[cfg(test)]
 mod tests {
```

Comment:
Wdyt of the following alternative? 

```suggestion
#![cfg(test)]
```

Reply:
Good idea, done + added `--all-targets`